data/fixtures/scopes/r/functionName.scope

```diff
@@ -0,0 +1,19 @@
+abc <- function(arg){
+    TRUE
+}
+---
+
+[Content] =
+[Domain] = 0:0-0:3
+  >---<
+0| abc <- function(arg){
```

Comment:
This is the only case that I'm unsure about and is likely to need review, I'm wondering if it would make more sense to include the operator in this for removal and insertion? Although I might struggle a little bit with adding them as custom delimiters

Reply:
`<-` should probably be added as a trailing range. That way it will be deleted if you delete the name.
We do the same thing with `=` in `value = 2`